incubator/kube-state-metrics/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: Install kube-state-metrics to generate and expose cluster-level metrics
+name: kube-state-metrics
+version: 0.3.0
```

Comment:
Just an idea, but I like using the Debian versioning standard; 

`${UPSTREAM_VERSION}-${BUILD_NUMBER}`

So, this would become

`0.3.0-1`

See https://www.debian.org/doc/manuals/debian-faq/ch-pkg_basics.en.html#s-pkgname